### PR TITLE
[GDPR] Adjusted TOS/Privacy acceptance

### PR DESCRIPTION
**Pull request by vincentsalucci**
## Code Changes
- **RegisterPage.xaml**: Added UI for terms/privacy acknowledgement. Created formatted label text with spannable selectors for each of the policies. Use specialized margin based on platform.
- **RegisterPageViewModel.cs**: Added variables for `ShowTerms`, `AcceptPolicies`, and created `PoliciesClickCommand` for opening URIs. Added `Submit` logic. Added getter property to handle switch margin based on platform.
- **AppResources.resx**: Added strings. Note: formatting for the `AcceptPolicies` string looks weird, but is necessary because of the intentional line break (xml:preserve takes the string as is).

## Screenshots
mobile-android-0-base
mobile-android-1-error

### CODE REVIEW

**Comment by kspearrin** [comment]
@mportune-bw to review

**Review by mportune-bw — COMMENTED** [review]
Looks good, awaiting some layout tweaks we discussed in chat

File: src/App/Pages/Accounts/RegisterPageViewModel.cs [hunk 1]
```diff
@@ -56,6 +56,13 @@ public bool AcceptPolicies
             set => SetProperty(ref _acceptPolicies, value);
         }
         
+        public Thickness GetSwitchMargin
```
Comment (mportune-bw) [inline src/App/Pages/Accounts/RegisterPageViewModel.cs:59]: This is a property so the name should just be `SwitchMargin`

Comment (vincentsalucci) [reply src/App/Pages/Accounts/RegisterPageViewModel.cs]: Updated.

**Review by mportune-bw — APPROVED** [review]

**Comment by vachan-maker** [comment]
@vincentsalucci When black theme is enabled , the color of the TOS and Privacy Policy links look odd. Can you take a look?Thanks

**Pull request merged**